C-Plus-Plus/README.md

```diff
@@ -132,6 +133,11 @@ _add list here_
 
 ## Other
 
+- [Array Right Rotation Naive Approach](cp/Array_Right_Rotate.cpp)
```

Comment:
Can we please remove the unnecessary changes from here?

Reply:
Done